Jenkinsfile

```diff
@@ -167,6 +167,8 @@ def initializeEnvironment() {
     sudo apt-get install socat
     pip install --upgrade pip
     pip install -U setuptools
+    sed -i -e 's/pyyaml.*/pyyaml/' ${HOME}/ccm/requirements.txt
```

Comment:
This line actually shouldn't be required.  What's in setup.py will be used when you do the "pip install" below.